docs/software/bittorrent.md

```diff
@@ -37,7 +33,445 @@
+
+## rTorrent - BitTorrent server with ruTorrent web interface
+
+rTorrent is a BitTorrent server. ruTorrent is also installed to allow for web interface access and control.  
+
+Also installs:
+
+- Webserver
+
+??? hint "USB dedicated hard drive highly recommended"
+
+![DietPi download software rTorrent](../assets/images/dietpi-software-download-rtorrent.png){: style="width:500px"}
+
+=== "Access to the web interface"
+
+    - URL = `http://<your.IP>/rutorrent`
+    - Username = `root`
+    - Password = `dietpi`
+
+=== "Access Downloads"
+
+    Make sure you have one of DietPi's [File Servers](https://dietpi.com/docs/software/file_servers/) installed.  
+    Folders used by rTorrent:
+
+    - folder accessed via file server = `downloads`
+    - local folder = `/mnt/dietpi_userdata/downloads`
+
+    Important note:  
+    rTorrent creates a lock file based on the systems hostname. When the hostname is changed (e.g. via `dietpi-config` \> ? Security Options`), then the old lock file needs to be removed to allow rTorrent creating a new one based on the new hostname:
+
+    ```sh
+    rm /mnt/dietpi_userdata/downloads/.session/rtorrent.lock
+    systemctl restart rtorrent
+    ```
+
+=== "Recommended: Protect your privacy with a VPN"
+
+    Although we enable forced encryption on all our BitTorrent clients, if you wish to ensure complete privacy and piece of mind for all your downloaded content, using a VPN is critical.  
+    We highly recommend [***NordVPN***](../vpn/#dietpi-nordvpn-openvpn-and-dietpi-gui-for-nordvpn-users) as it offers unlimited bandwidth, zero logging and up to 6 devices on a single account.  
+    ![DietPi download software VPN NordVPN](../assets/images/dietpi-software-vpn-nordvpn-logo.png){: style="width:200px"}
+
+## Aria2 - Download manager with web interface
+
+Download manager, with a slick web UI interface. Also supports BitTorrent downloads.
+
+Also installs:
+
+- Webserver
+
+![DietPi download software Aria2](../assets/images/dietpi-software-download-aria2.png){: style="width:500px"}
+
+=== "Access to the web interface"
+
+    - URL = `http://<your.IP>/aria2/docs`
+    - rpc secret password = `dietpi`
+
+    This needs to be entered in the following location, in the web interface:
+
+    - `Settings`
+    - `Connection Settings`
+    - Enter the rpc secret password into the `Enter the secret token (optional):` box, then save changes
+
+=== "Access Downloads"
+
+    Make sure you have one of DietPi's [File Servers](https://dietpi.com/docs/software/file_servers/) installed.  
+    Folders used by Aria2:
+
+    - folder accessed via file server = `downloads`
+    - local folder = `/mnt/dietpi_userdata/downloads`
+
+=== "Make setting changes permanently"
+
+    Please note, the `aria2-webui` does not support saving settings after session shutdown, this is a known limitation with the software, please use the file `aria2.conf` to make changes:
+
+    - Stop services with `dietpi-services stop`
```

Comment:
Let's suggest to start/stop the `aria2` service only:
```
systemctl stop aria2
```

Reply:
Ok.